editor-extensions/vscode/server/src/server.ts

```diff
@@ -119,9 +128,10 @@ connection.onInitialize((params: InitializeParams) => {
 	const result: InitializeResult = {
 		capabilities: {
 			textDocumentSync: TextDocumentSyncKind.Incremental,
-			codeLensProvider: {
-				resolveProvider: true
-			}
+			// codeLensProvider: {
+			// 	resolveProvider: true
+			// },
```

Comment:
Can probably just delete this if it's not needed anymore.
```suggestion
```

Reply:
Just added some comments - it'll be coming back once LSP supports it.  Happy to delete it for now though to tidy up the code.